tests/core/manifest/TemplateLoaderTest.php

```diff
@@ -26,6 +26,8 @@ public function setUp() {
 	 * Test that 'main' and 'Layout' templates are loaded from module
 	 */
 	public function testFindTemplatesInModule() {
```

Comment:
Should we delete these tests? Refactor them? Mark them as skipped or incomplete?